hack/examples/patches/metallb.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 Nutanix. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+- op: "add"
+  path: "/spec/topology/variables/0/value/addons/serviceLoadBalancer"
+  value:
+    provider: MetalLB
+    configuration:
+      addressRanges:
+        - start: 198.18.1.1
```

Comment:
i see that you have these in an order that makes sense here, but kustomize reverses the order. a little frustrating, but thanks for trying to make it more readable.

Reply:
Yeah, I noticed that, too.

 I started off by modifying the cluster template directly. I didn't know that we generated the example cluster templates from `hack/examples`. Now I know, and I'll document it in a separate PR :smile: